test/wdio/form-associated/cmp.tsx

```diff
@@ -5,8 +5,8 @@ import { AttachInternals, Component, h } from '@stencil/core';
   formAssociated: true,
 })
 export class FormAssociatedCmp {
-  @AttachInternals()
-  internals: ElementInternals;
+  // @ts-ignore typescript complains because this isn't initialized
```

Comment:
Should we tweak the `tsconfig.json` for this?

Reply:
yeah I think that's a good idea, I turned `strictPropertyInitialization` off